src/components/AWSLambdaPageTable/AWSLambdaPageTable.tsx

```diff
@@ -17,12 +17,12 @@ import React, { FC, useContext, useEffect, useState } from 'react';
 import { Typography, Box, Button } from '@material-ui/core';
 import GitHubIcon from '@material-ui/icons/GitHub';
 import { Table, TableColumn } from '@backstage/core';
-import { useEntityCompoundName } from '@backstage/plugin-catalog';
+import { Entity } from '@backstage/catalog-model';
+import moment from 'moment';
```

Comment:
No need to block this commit over it, but would you mind replacing `moment` with something else please? The project is in maintenance mode and there are far smaller libs out there which will do the job we need. e.g. [date-fns](https://date-fns.org/).